lerna.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+  "useWorkspaces": true,
+  "version": "independent",
+  "command": {
+    "publish": {
+      "conventionalCommits": true,
+      "message": "chore: Release packages [skip ci]"
```

Comment:
We will need this to skip ci when a package is released. This is just an example